cmd/clusterctl/client/tree/discovery_test.go

```diff
@@ -110,7 +110,7 @@ func Test_Discovery(t *testing.T) {
 			name: "Discovery with grouping disabled",
 			args: args{
 				discoverOptions: DiscoverOptions{
-					DisableGrouping: true,
+					Grouping: true,
```

Comment:
I was expecting this to be 
```suggestion
					Grouping: false,
```
Given that the the new flag is semantically the opposite of the old one.
Same below in other tests

Reply:
okay updating it

Reply:
Updated it